services/src/test/java/io/scalecube/services/ServiceUnitTest.java

```diff
@@ -0,0 +1,134 @@
+package io.scalecube.services;
+
+import static org.junit.Assert.assertTrue;
+
+import io.scalecube.services.ServicesConfig.Builder.ServiceConfig;
+import io.scalecube.services.annotations.AnnotationServiceProcessor;
+import io.scalecube.services.annotations.ServiceProcessor;
+import io.scalecube.services.routing.RandomServiceRouter;
+import io.scalecube.services.routing.Router;
+import io.scalecube.services.routing.RouterFactory;
+import io.scalecube.transport.Address;
+import io.scalecube.transport.Message;
+
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Optional;
+
+public class ServiceUnitTest {
```

Comment:
Typically for unit tests the convention is `<Class name you test>Test`, also I see in this case there are several tests not really related to each other. So maybe create several test classes where each will correspond to specific class you test or give some meaningful name to the test to display the fact why those tests are collocated in single file and remove word `Unit` from the name.